### DISCUSSION TITLE
ModuleNotFoundError: No module named 'dotenv'

### DISCUSSION THREAD

**Discussion by MoonTucker** [imartinez/privateGPT#555] [General]
When I try to get to the query prompt by entering 'python ingest.py', I encounter this:

"C:\Users\equus\OneDrive\Desktop\PrivateGPT\PrivateGPT\ingest.py", line 5, in <module>
    from dotenv import load_dotenv
<img width="413" alt="error" src="https://github.com/imartinez/privateGPT/assets/134952550/3d81c120-9105-43a9-90f7-10959032a230">

**Expected behavior**
I expected to enter a query

**Environment (please complete the following information):**
 - OS / hardware: windows 11
 - Python version [e.g. 3.10.6
<img width="413" alt="image" src="https://github.com/imartinez/privateGPT/assets/134952550/ab7b1bce-83ad-4f4d-bbf6-7a26aed58022">

**Comment by kvchitrapu** [comment]
I had similar issue on Mac. I resolved it by setting PYTHONPATH like this 
```
python -m venv venv
source venv/bin/activate
pip3 install -r requirements.txt
export PYTHONPATH=${VIRTUAL_ENV}/lib/python3.11/site-packages/:${PYTHONPATH}
python ingest.py
```

**Reply by tounsicorp** [reply]
Got a guess what to do on PC? :D

**Reply by Ciaranwuk** [reply]
create a virtual environment (lots of options and instructions online for this, I personally use mamba but kvchitrapu has used a pip package called venv) , and reinstall everything. Hopefully you won't even need to do the PYTHONPATH command, but if you do still have the problem, then try doing that same command
